packages/analytics/analytics-chart/src/components/VueJsonCsv.vue

```diff
@@ -18,7 +18,19 @@ import { unparse } from 'papaparse'
 
 const emit = defineEmits(['export-started', 'export-finished'])
 
-const isType = (value: any, type: string) => typeof value === type
+enum ValidType {
+  String = 'string',
+  Number = 'number',
+  Boolean = 'boolean',
+  Object = 'object',
+  Function = 'function',
+  Undefined = 'undefined',
+  Symbol = 'symbol'
+}
+
+const isType = (value: any, type: ValidType) => {
+  return typeof value === type
```

Comment:
@filipgutica this is the line giving me trouble, even though I'm using that enum up above

`Typeof comparisons should be to string literals`